packages/hoppscotch-common/src/modules/dioc.ts

```diff
@@ -3,8 +3,10 @@ import { Container, ServiceClassInstance } from "dioc"
 import { diocPlugin } from "dioc/vue"
 import { DebugService } from "~/services/debug.service"
 import { platform } from "~/platform"
+import { ScrollService } from "~/services/scroll.service"
 
 const serviceContainer = new Container()
+serviceContainer.bind(ScrollService)
```

Comment:
we do not need to bind this, we can remove this line. You can directly use the service with `useService` call.